.github/workflows/check-lighthouse-schema.yml

```diff
@@ -0,0 +1,54 @@
+name: Check schema
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - .github/workflows/*lighthouse*.yml
+      - api/**
```

Comment:
Should this be as general as that? I'm not sure what can affect the generated schema, but happy to be more specific if you point me toward the best glob pattern :)

Reply:
I'd be fine to leave it like this.  The API project doesn't change too often and I'd rather have it too general than too specific and miss things.